cmd/snap/main.go

```diff
@@ -92,6 +94,17 @@ type parserSetter interface {
 // Since commands have local state a fresh parser is required to isolate tests
 // from each other.
 func Parser() *flags.Parser {
+	optionsData.Version = func() error {
```

Comment:
I think it also works if I just make it a `func()` returning nothing. I'll check.